content/Video_Display.md

```diff
@@ -553,27 +553,28 @@ the one at `0:9800`!
 ### Background (BG)
 
 The [SCY and SCX](#ff42-scy-scroll-y-r-w-ff43-scx-scroll-x-r-w) registers can be
-used to scroll the background, allowing to select the origin of the visible
-160x144 pixel area within the total 256x256 pixel background map.
+used to scroll the Background, specifying the origin of the visible
+160x144 pixel area within the total 256x256 pixel Background map.
 The Background visible area wraps around the Background map (that is, when part of
 the visible area goes beyond the map edge, it starts displaying the opposite side of the map).
 
-Whether the background is displayed can be toggled using
-[LCDC bit 0](#lcdc-0-bg-window-display-priority), except on CGB in CGB Mode,
-where it's always drawn.
+In Non-CGB mode, the Background (and the Window) can be disabled using
+[LCDC bit 0](#lcdc-0-bg-window-display-priority).
 
 ### Window
 
 Besides the Background, there is also a Window overlaying it.
 The content of the Window is not scrollable; it is always
-displayed starting at the top left tile of its tilemap. The only way to adjust the Window
+displayed starting at the top left tile of its tile map. The only way to adjust the Window
```

Comment:
@avivace We should probably standardize on "tilemap" vs. "tile map" in the style guide.

Reply:
The problem is that the word "tilemap" doesn't exist. I wouldn't be used in a book.

Reply:
Neither does "VBlank" :P Though I'd favor "tile map", anyway.

Reply:
+1 for tile map. Adding it to the wiki.